common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
```

Comment:
Not sure what's up with this, could you explain? Looks like a bunch of the pattern library styles just copied in to a new folder? We should just be using the real pattern library directly, if possible - it's installed in Platform already.